index.html

```diff
@@ -152,5 +153,49 @@ <h5>Skills</h5>
           </ul>
       </div>
     </section>
+    <section class="contact-form">
+      <p>
+        I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+      </p>
+      <form action="https://formspree.io/f/mnqwdall" method="post">
```

Comment:
- [ ] In Original website, the place holder and the input field on the left side , please change according to original website.
- Your website 
![ddasdasdas](https://user-images.githubusercontent.com/26767383/153953577-8c803104-f3ce-4745-abcf-22a1c0ab36af.PNG)

- Original website
![wefwefdsc](https://user-images.githubusercontent.com/26767383/153953607-d0d25135-a851-4ccf-ac88-b2818e003316.PNG)